KubeArmor/core/kubeUpdate.go

```diff
@@ -127,6 +127,11 @@ func (dm *KubeArmorDaemon) UpdateContainerGroupWithPod(action string, pod tp.K8s
 			dm.RuntimeEnforcer.UpdateSecurityProfiles(action, pod, true)
 		}
 
+		if pod.AlreadyDeployed {
```

Comment:
I think we don't need this

Reply:
We do need this one only without the condition that is. Its because if we remove this the already deployed container logs wont be able to get matched because their MatchPolicy struct wont get added if added policy event was received earlier than this function call. The reason is that for already deployed pods there is only ADDED event and no MODIFIED event. Our code today calls LogFeeder.UpdateSecurityPolicy only inside MODIFIED case.